src/getting-started.md

```diff
@@ -1,10 +1,16 @@
 # Getting started
 
-This section is designed to help you integrate the FIB API seamlessly into your application or system by providing instructions on how to register an account, check the API availability, call API endpoints, and test your implementation in a sandbox environment.
+Since FIB allows its users to interact with the network both via API and the Web App, this section provides instructions for all necessary preliminary steps, be you a regular user or a developer aiming to integrate the FIB API into their application or system.
```

Comment:
```suggestion
Since FIB allows its users to interact with the network both via API and the Web App, this section provides instructions for all necessary preliminary steps, be you a regular user or a developer aiming to integrate the FIB API into their application or system.
```
```suggestion
Since FIB allows its users to interact with the network both via API and the Web App, this section provides instructions for all necessary preliminary steps, be you a frontend application user or a developer aiming to integrate the FIB API into their application or system.
```